DB Setup/createDB.sql

```diff
@@ -10,6 +10,7 @@ create table SessionKeys (
 
```

Comment:
Can this folder be renamed to not have a space in it?

Reply:
+1

Reply:
Done :)